GlobalRegistry.gd

```diff
@@ -392,8 +392,9 @@ func getCharacter(id: String):
 		return GM.pc
 	
 	if(!characters.has(id)):
-		Log.printerr("ERROR: character with the id "+id+" wasn't found")
+		if (id != ""): Log.printerr("ERROR: character with the id "+id+" wasn't found")
```

Comment:
Would rather makes this into `Log.printerr("ERROR: character with the id "+str(id)+" wasn't found")` (without the if)
That way it would show if we trying to use an empty-string or a null character rather than omitting it